### ISSUE TITLE
Errors when running test suite when using ruby versions other than 2.2.1

### ISSUE THREAD

**Issue by c1505** [learn-co-curriculum/oo-student-scraper#8]
I had errors when running locally on a linux machine using ruby 2.2.2 and then also on nitrous that uses 2.1.5.  The tests worked on Cloud9 using version 2.2.1.  Switched my local machine to Ruby 2.2.1, reinstalled the gems, and the tests ran.  Have not been able to get it working in nitrous still.

**Comment by c1505** [comment]
'rspec spec/student_spec.rb
/usr/local/opt/rbenv/versions/2.1.5/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel
_require.rb:54:in `require': cannot load such file -- spec (LoadError)
        from /usr/local/opt/rbenv/versions/2.1.5/lib/ruby/site_ruby/2.1.0/rubygems/co
re_ext/kernel_require.rb:54:in `require'
        from /home/nitrous/code/labs/oo-student-scraper-v-000/.bundle/gems/webmock-1.
22.3/lib/webmock/rspec.rb:15:in `rescue in <top (required)>'
        from /home/nitrous/code/labs/oo-student-scraper-v-000/.bundle/gems/webmock-1.
22.3/lib/webmock/rspec.rb:10:in `<top (required)>'
        from /usr/local/opt/rbenv/versions/2.1.5/lib/ruby/site_ruby/2.1.0/rubygems/co
re_ext/kernel_require.rb:54:in `require'
        from /usr/local/opt/rbenv/versions/2.1.5/lib/ruby/site_ruby/2.1.0/rubygems/co
re_ext/kernel_require.rb:54:in `require'
        from /home/nitrous/code/labs/oo-student-scraper-v-000/spec/spec_helper.rb:6:i
n `<top (required)>'
        from /usr/local/opt/rbenv/versions/2.1.5/lib/ruby/site_ruby/2.1.0/rubygems/co
re_ext/kernel_require.rb:54:in `require'
        from /usr/local/opt/rbenv/versions/2.1.5/lib/ruby/site_ruby/2.1.0/rubygems/co
re_ext/kernel_require.rb:54:in `require'
        from /usr/local/opt/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/rspec-core-
3.4.1/lib/rspec/core/configuration.rb:1295:in `block in requires='
        from /usr/local/opt/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/rspec-core-
3.4.1/lib/rspec/core/configuration.rb:1295:in `each'
        from /usr/local/opt/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/rspec-core-
3.4.1/lib/rspec/core/configuration.rb:1295:in `requires='
        from /usr/local/opt/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/rspec-core-
3.4.1/lib/rspec/core/configuration_options.rb:109:in `block in process_options_into'

**Comment by AnnJohn** [comment]
Hi @c1505 sorry about this. It looks like nitrous is still running ruby 2.1.5 instead of your updated version. This might be an issue specific to you (we'll also look at the lab on our end), but could you please use the Ask a Question tool in Learn and someone from our support team can help you one-on-one? Thanks!

**Comment by c1505** [comment]
I did @AnnJohn .  Sam helped me get it working with that change in ruby versions.  I prefer to use my linux machine over nitrous anyways so it isn't an issue for me.  I only tried that route because it didn't work locally for me initially and nitrous is officially supported.  

I just wanted to document it here in case someone else had the same problem.  If it really ends up being just me, this can be closed.
